packages/dom/src/feature-app-loader.ts

```diff
@@ -0,0 +1,56 @@
+import {FeatureAppManager} from '@feature-hub/core';
+import {LitElement, html, property} from 'lit-element';
+import {TemplateResult} from 'lit-html';
+import {until} from 'lit-html/directives/until';
+import {createFeatureAppContainer} from './feature-app-container';
+
+export function createFeatureAppLoader(
+  manager: FeatureAppManager
+): typeof HTMLElement {
+  createFeatureAppContainer(manager);
+
+  class FeatureAppLoader extends LitElement {
+    @property({type: String})
+    public src = '';
+
+    @property({type: String})
+    public idSpecifier: string | undefined;
+
+    public render(): TemplateResult {
+      return html`
+        ${until(
+          this.loadFeatureApp(),
+          html`
+            <slot name="loading"></slot>
+          `
+        )}
+      `;
+    }
+
+    private async loadFeatureApp(): Promise<TemplateResult> {
+      try {
+        const definition = await manager.getAsyncFeatureAppDefinition(this.src)
+          .promise;
+
+        return html`
+          <feature-app-container
```

Comment:
Have you thought about using https://github.com/phaux/babel-jsx-lit-html? I understand that in terms of CI the feature can also come later. Maybe we need an issue for that?